packages/amplify_authenticator/lib/src/l10n/generated/input_localizations_en.dart

```diff
@@ -126,7 +126,7 @@ class AuthenticatorInputLocalizationsEn
   }
 
   @override
-  String passwordRequirementsAtLeast(int numCharacters, String characterType) {
+  String passwordRequirementsAtLeast(num numCharacters, String characterType) {
```

Comment:
Is this not always an int? I assume if it is not an int it is still a whole number?

Reply:
This was changed on one of my initial commits, and I cannot recall why. I'm changing it back. If there is an issue with it, it's immaterial to this PR.